src/js/navigation.js

```diff
@@ -2,12 +2,54 @@ const navigationToggle = document.querySelector(".menu-btn");
 const navigation = document.querySelector("nav");
 const welcome = document.querySelector(".welcome");
 
-navigationToggle.addEventListener("click", ()  => {
+navigationToggle.addEventListener("click", () => {
   document.body.classList.toggle("nav-is-open");
```

Comment:
It is enough that you just add `nav-is-open` class to the body, and you can your `css` based on this. You don't need to toggle also `"welcome-to-right`

Reply:
z dodajanjem welcome-to-right lahko izberem področje izven nav, na katerega obesim event listener